Palo Alto Networks/paloalto-cortex/_meta/smart-descriptions.json

```
@@ -0,0 +1,33 @@
+[
+  {
+    "value": "{cortexxdr.category}: An event of {cortexxdr.severity_human} severity was {cortexxdr.action} on endpoint {cortexxdr.shost}, initiated by user {cortexxdr.suser.0}.",
```

According to the suggestion in the parser:
```suggestion
    "value": "{cortexxdr.category}: An event of {cortexxdr.severity_human} severity was {event.action} on endpoint {host.name}, initiated by user {user.domain}\\{user.name}.",
```